website/content/docs/integrations/consul-integration.mdx

```diff
@@ -93,16 +262,45 @@ remaining fields are required to match those shown here.
 }
 ```
 
-</Tab>
-<Tab heading="Consul Enterprise">
+</CodeBlockConfig>
+
+The `JWKSURL` address must be reachable by all Consul servers and should
+resolve to multiple Nomad agents to avoid a single point of failure. Both Nomad
+servers and clients are able to handle this request.
+
+[![Consul Auth Method][img_consul_auth_method]][img_consul_auth_method]
+
+When Nomad is configured to use mTLS, [`tls.verify_https_client`][] must be set
+to `false` since it is not possible to provide client certificates to the
+Consul auth method. Alternatively, you may expose Nomad's JWKS URL from a proxy
+or load balancer that terminates the TLS connection.
+
+If the Consul servers are not able to reach Nomad's JWKS URL, the public keys
+may be read from Nomad's `/.well-known/jwks.json` and provided to the auth
+method directly using the [`JWTValidationPubKeys`][] parameter. The keys must
+be converted from JWKS to PEM format and **are rotated periodically**, so this
+process must be done continually. The rotation frequency is controlled by the
+[`server.root_key_rotation_threshold`][] configuration of the Nomad servers.
+
+When an allocation that needs access access to Consul starts, the Nomad client
+running it exchanges the Nomad workload identities for Consul from tasks and
+services for Consul ACL tokens.
```

Comment:
It's clear from context that these are the WIs for Consul services and leaving it makes this sentence a little hard to parse (with "for Consul" being in there twice).

```suggestion
When an allocation that needs access access to Consul starts, the Nomad client
running it exchanges the Nomad workload identities for tasks and services for 
Consul ACL tokens.
```

Reply:
I also had `access` twice in there 😅